styles/form.styles.css

```diff
@@ -0,0 +1,114 @@
+.form-section {
+  height: 120vh;
+  background-color: #6070ff;
+  background-image: url("../images/svg/form-bg-mob.svg");
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  border-top-left-radius: 100px;
+}
+
+.container-form {
+  width: var(--mobile-width);
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 20px;
+}
```

Comment:
- [ ] Great job you did here 👏 . However in comparison to the template you chose, lets make a few adjustments
- Kindly shift the `submit button` to the beginning of the message field as shown on the template
- The background images are not placed properly on your design. I've circled the missing parts from the template in red so you can see the images missing in yours.
- [ ] The goal is to have you build a close replica of the original design, which will be same for future managers or clients 👍 

## Template:
![image](https://github.com/DHE19/portfolio-microverse/assets/50990925/53dffc88-0768-4c06-a171-31dcbca642d8)

## Your work:
![image](https://github.com/DHE19/portfolio-microverse/assets/50990925/9a407ab2-f1a7-42e2-a2be-5a2155341d3e)